bitcoin/rpc.py

```diff
@@ -654,6 +654,17 @@ def validateaddress(self, address):
             r['pubkey'] = unhexlify(r['pubkey'])
         return r
 
+    def unlockwallet(self, password, timeout=60):
+    	"""Stores the wallet decryption key in memory for 'timeout' seconds.
```

Comment:
One last nit: you have both tabs and spaces at the beginning of this line, which is incorrect.